crates/cheatnet/tests/cheatcodes/deploy.rs

```diff
@@ -315,3 +315,37 @@ fn deploy_invokes_constructor() {
 
     assert_success!(output, vec![Felt252::from(123)]);
 }
+
+#[test]
+fn deploy_at_invokes_constructor() {
```

Comment:
Is there any way to handle panic in constructor? If yes please add a test for asserting it.

Reply:
[Add a test case checking for the correct constructor error handling f…](https://github.com/foundry-rs/starknet-foundry/pull/828/commits/18a416ded39029313939bf492306d047d93d11cd)